src/components/CourseForm/CourseForm.jsx

```diff
@@ -38,12 +36,12 @@ function CreateCourse() {
 
     dispatch(
       addCourse({
-        id: createGuid(),
         title: event.target.elements.title.value,
         description: event.target.elements.description.value,
-        creationDate: formatCreationDate(),
-        duration: event.target.elements.duration.value,
-        authors: courseAuthors,
+        duration: Number(event.target.elements.duration.value),
+        authors: courseAuthors.map((author) => {
+          return author.id;
+        }),
```

Comment:
1. You can just use `authors: courseAuthors.map((author) => author.id)`
2. Since you updated the value, name does not match it anymore. Let's rename it also to `authorIds`